### ISSUE TITLE
"title" and "placeholder" not consistently supported for name calculation

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/html-api-map#5]
while on windows (testing briefly with jaws and nvda and various browsers - chrome, firefox, IE11, edge) it seems that `placeholder` and `title` are indeed used (when nothing else is present) to provide a name for a control (e.g. an `input`), this is not currently the case everywhere. just doing a quick test on mobile:

- on iOS/Safari/VoiceOver, once the user enters anything into an `input` named only with a `placeholder`, the placeholder text is not announced anymore (so you just get "input - whatever the user typed"; `title` does work (even when the user entered something, it is announced)
- on Android/Chrome/TalkBack, in both cases, as soon as the user entered something into the `input`, neither `placeholder` nor `title` are announced anymore

not tested on OS X, but I'll hazard a guess it behaves the same as iOS/VO

question now is: are these browsers wrong and should bugs be filed against them? or should `placeholder` and `title` be removed from the name calculation here? http://rawgit.com/w3c/html-api-map/master/index.html#accessible-name-and-description-calculation /cc @stevefaulkner

**Comment by jasonkiss** [comment]
This issue was moved to w3c/html-aam#84
